qa-heal.planx-pla.net/metadata/aggregate_config.json

```diff
@@ -463,63 +480,72 @@
                     "medicaid-funded-mat-for-justice-involved-persons-1565019237",
                     "medication-assisted-treatment-with-methadone-mat-laws",
                     "recreational-marijuana-laws"
-				]
-			},
-			"keep_original_fields": false,
-			"field_mappings": {
-				"tags": [],
-				"authz": "",
-				"sites": "",
-				"summary": {
-					"path":"description",
-					"filters": ["strip_html"]
-				},
-				"study_description_summary": {
-					"path":"description",
-					"filters": ["strip_html"]
-				},
-				"location": "United States",
-				"subjects": "",
-				"__manifest": "",
-				"study_name": "",
-				"study_name_title": "",
-				"study_type": "",
-				"institutions": "Center for Public Health Law Research at the Temple University Beasley School of Law",
-				"year_awarded": "",
-				"investigators": "",
-				"investigators_name": "",
-				"project_title": "path:title",
-				"protocol_name": "",
-				"study_summary": { "path":"description", "filters": ["strip_html"] },
-				"_file_manifest": "",
-				"dataset_1_type": "",
-				"dataset_2_type": "",
-				"dataset_3_type": "",
-				"dataset_4_type": "",
-				"dataset_5_type": "",
-				"project_number": "path: id",
-				"dataset_1_title": "",
-				"dataset_2_title": "",
-				"dataset_3_title": "",
-				"dataset_4_title": "",
-				"dataset_5_title": "",
-				"administering_ic": "NIDA",
-				"advSearchFilters": [],
-				"dataset_category": "",
-				"research_program": "",
-				"research_question": "",
-				"study_description": "Observational",
-				"clinical_trial_link": "",
-				"dataset_description": "",
-				"research_focus_area": "",
-				"dataset_1_description": "",
-				"dataset_2_description": "",
-				"dataset_3_description": "",
-				"dataset_4_description": "",
-				"dataset_5_description": "",
-				"data_availability": "pending"
-			},
-			"per_item_values": {
+                ]
```

Comment:
is that the indentation correction here? if yes, thanks!

Reply:
Yes it is